src/data/veilederbrukerknytning/useTildelVeileder.ts

```diff
@@ -0,0 +1,41 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { useValgtEnhet } from "@/context/ValgtEnhetContext";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { SYFOOVERSIKTSRV_PERSONTILDELING_ROOT } from "@/apiConstants";
+import { post } from "@/api/axios";
+import { veilederBrukerKnytningQueryKeys } from "@/data/veilederbrukerknytning/useGetVeilederBrukerKnytning";
+
+export const useTildelVeileder = () => {
+  const personident = useValgtPersonident();
+  const { valgtEnhet } = useValgtEnhet();
+  const queryClient = useQueryClient();
+
+  const path = `${SYFOOVERSIKTSRV_PERSONTILDELING_ROOT}/registrer`;
+  const postTildelVeileder = (veilederIdent: string) => {
+    const veilederBrukerKnytning: VeilederBrukerKnytning = {
+      veilederIdent: veilederIdent,
+      fnr: personident,
+      enhet: valgtEnhet,
+    };
+
+    return post(path, {
```

Comment:
APIet som finnes i `syfooversiktsrv` tar inn en liste med `tilknytninger` (siden man i oversikten kan velge flere personer). I tillegg er `enhet: string` påkrevd uten at den brukes til noe i APIet. Vi kan vurdere å lage et eget endepunkt i `syfooversiktsrv` (uten liste og enhet) som bare brukes herfra. Da kan vi også enklere telle i grafana hvor mange tildelinger som skjer fra `syfomodiaperson`.

Reply:
Høres ut som noe vi kan gjøre ja, men evt forbedre etterhvert.

Hvis enhet hadde vært mulig å sende inn og at den faktisk ble brukt til noe, så har vi vel løst hele "ruting" problematikken internt hos oss i syfo? 😬 Så vi er jo åpenbart ikke langt unna å løse noe sånt, men det er en veldig lokal løsning på et større problem da.

Reply:
Vi hadde et ubrukt endepunkt i `syfooversiktsrv` så tok og ryddet litt der nå. Fikser eget api for syfomodiaperson samtidig, kommer PR :)